kernel/packages/decentraland-loader/lifecycle/controllers/scene.ts

```diff
@@ -26,49 +25,75 @@ export class SceneLifeCycleController extends EventEmitter {
     this.enabledEmpty = opts.enabledEmpty
   }
 
-  contains(status: SceneLifeCycleStatus, position: Vector2Component) {
-    return (
-      status.sceneDescription &&
-      status.sceneDescription.sceneJsonData.scene.parcels.includes(`${position.x},${position.y}`)
-    )
+  async reportSightedParcels(sightedParcels: string[], lostSightParcels: string[]) {
```

Comment:
Most changes to this file were refactors. The only thing that added was the `reloadScene` method. The refactors were:
1. We deleted methods `hasStarted` and `contains` that were not being used
1. Renamed `onSight` to `startSceneLoading`
1. Renamed `lostSight` to `unloadScenes`
1. Set a lot of methods to private (since they weren't being used outside this class), and moved them to the bottom of the file